tests/hpc/slurm_master.pm

```diff
@@ -225,10 +225,10 @@ sub t01_accounting() {
     my $description = 'Basic check for slurm accounting cmd';
     my $result = 0;
     my %users = (
-        'user_1' => 'Sebastian',
```

Comment:
keeping upper case imo makes sense as slurm should handle that and this is a test per se - to see if indeed users will be added to the db with lower cases